gfg_interactive/exp/templates/category_switch/results.html

```diff
@@ -1,8 +1,10 @@
-{% extends "base.html" %}
-{% block content %}
+{% if not content_only %}
```

Comment:
This is a nice and targeted fix- good use of an undocumented (but pre-existing) feature!